lib/components/user/notification-prefs-pane.js

```diff
@@ -85,13 +226,59 @@ class NotificationPrefsPane extends Component {
             </FormGroup>
           )}
           {notificationChannel === 'sms' && (
-            <FormGroup>
-              <ControlLabel>Enter your phone number for SMS notifications:</ControlLabel>
-              {/* TODO: Add field validation. */}
-              <FormControl onChange={this._handlePhoneNumberChange} type='tel' value={phoneNumber} />
-            </FormGroup>
+            <div>
+              <FormGroup validationState={phoneFieldValidationState}>
+                <ControlLabel>Enter your phone number for SMS notifications:</ControlLabel>
+                {/* TODO: Add field validation. */}
+
+                <FormControl
+                  onChange={this._handlePhoneNumberChange}
+                  placeholder='+12223334567'
```

Comment:
This is actually a valid phone number, so we shouldn't use it. Instead use something with a 555 area code and 555 prefix. [Read here for more info](https://en.wikipedia.org/wiki/555_(telephone_number)).

Reply:
Good to know! Fixed in https://github.com/opentripplanner/otp-react-redux/pull/224/commits/626fdb7a50e90c7a827c98e3d9a00d29f2025365.